modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -190,10 +194,19 @@ The name, icon, and description properties can only be changed in the Exchange a
 If you have contributor or admin access to an asset, you can add a new version from the asset portal:
```

Comment:
There is also another way to add a version through the versions table modal (manage versions button). It may be worth adding a note about that.